package.json

```diff
@@ -83,6 +83,7 @@
 		"@babel/core": "7.4.0",
 		"@babel/register": "7.4.4",
 		"@wordpress/browserslist-config": "2.5.0",
+		"@wordpress/components": "8.0.0",
```

Comment:
Because the dependency is externalized (https://github.com/Automattic/jetpack/pull/13631/files#r330871100) and pulled from global `window.wp.components` anyway, this might not be needed here. Can you try if it just works without this or does eslint complain?